std/stdio.d

```diff
@@ -346,8 +346,8 @@ Hello, Jimmy!
  */
 struct File
 {
-    import std.traits : isScalarType, isArray;
```

Comment:
Could you explain what you mean by the term "churn?"  I think this is a worthwhile style fix.